packages/scheduler/src/ScheduledTask.ts

```diff
@@ -0,0 +1,40 @@
+/** @license MIT License (c) copyright 2010-2017 original author or authors */
+import { ScheduledTask, Time, Task, Scheduler } from '@most/types'
+
+export default class ScheduledTaskImpl implements ScheduledTask {
+  /**
+   * @mutable
+   */
+  time: Time;
```

Comment:
Wow, do we really mutate this from outside the class?  If so, that's a bummer!  What was I thinking, lol 😄

Reply:
Yeah, here: https://github.com/mostjs/core/pull/296/files/23a9a0bdc747ac94e595309d28057770c864536e#diff-bf49420dee5ae6f33cc0598cf149d325R81

Reply:
I see: https://github.com/mostjs/core/blob/master/packages/scheduler/src/Timeline.js#L75

Again ... what was I thinking?  lol.  Fine to keep it as is, and hopefully we can figure out a better way later.